src/components/pools/whitelist/nft/NftCollection.tsx

```diff
@@ -145,37 +145,37 @@ const NftCollection = ({
               onNftDelete={onNftDelete}
               onNftIdChange={onNftIdChange}
               onNftMinimumAmountChange={onNftMinimumAmountChange}
-              selectedCollectionNftsIds={selectedCollection.nftCollectionData.nftsIds}
+              // TODO: Revisit nft ids
```

Comment:
Can you please merge epic branch?

Reply:
Already done. There's a Typescript issue on the NftsMinimumAmounts component.

Reply:
It says: `Property 'invalidNftIds' does not exist on type 'SelectedNftCollectionData'.` Can you take a look at it when you have some time?

Reply:
I mean, it's out of the scope of this PR, but can you create another PR to fix that ts issue?

Reply:
I don't understand what you mean. Seems like everything okay in this and epic branches, no?

Reply:
Forgot it. Just restarted my IDE and they disappeared